gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https://services.gradle.org/distributions/gradle-4.9-all.zip
+distributionUrl=https://services.gradle.org/distributions/gradle-6.9-all.zip
```

Comment:
There should be some other files changed, please use `./gradlew wrapper --gradle-version 6.9` to upgrade

Reply:
Done.